apps/src/lib/kits/maker/ui/SetupGuide.jsx

```diff
@@ -1,417 +1,109 @@
-import _ from 'lodash';
 import React from 'react';
-import yaml from 'js-yaml';
-import SetupChecklist from './SetupChecklist';
+import PropTypes from 'prop-types';
+import experiments from '@cdo/apps/util/experiments';
 import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
-import i18n from '@cdo/locale';
+import SetupInstructions from '@cdo/apps/lib/kits/maker/ui/SetupInstructions';
 import applabI18n from '@cdo/applab/locale';
-import {
-  isCodeOrgBrowser,
-  isOSX,
-  isWindows,
-  isLinux
-} from '../util/browserChecks';
-import Button from '../../../../templates/Button';
-import ToggleGroup from '../../../../templates/ToggleGroup';
-import FontAwesome from '../../../../templates/FontAwesome';
-import {CHROME_APP_WEBSTORE_URL} from '../util/makerConstants';
-import {createStore, combineReducers} from 'redux';
-import isRtl from '@cdo/apps/code-studio/isRtlRedux';
-import responsive from '@cdo/apps/code-studio/responsiveRedux';
-import {Provider} from 'react-redux';
-import experiments from '@cdo/apps/util/experiments';
-import {
-  WEB_SERIAL_FILTERS,
-  shouldUseWebSerial
-} from '@cdo/apps/lib/kits/maker/util/boardUtils';
-
-const DOWNLOAD_PREFIX = 'https://downloads.code.org/maker/';
-const WINDOWS = 'windows';
-const MAC = 'mac';
-const LINUX = 'linux';
-const CHROMEBOOK = 'chromebook';
+import i18n from '@cdo/locale';
 
 const style = {
-  icon: {
-    float: 'left',
-    padding: '5px'
+  main: {
+    width: '80%',
```

Comment:
From the screenshots, the development content looks much narrower than production content. Is this a concern?

Reply:
Test images retaken with same zoom.
Refer to testing story.